### DISCUSSION TITLE
Bellows exception

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#999] [Q&A] [answered]
Bellows radio lib is using the following 2 exceptions. Is there an equivalent at Zigpy lib. It is a shame that I have to import 
bellows.exception in my plugin code.

```
   bellows.exception.EzspError
   bellows.exception.ControllerError 
```

**Comment by puddly** [comment, marked as answer]
Both inherit from zigpy exceptions, you don't need to specifically catch the bellows subclasses: https://github.com/zigpy/bellows/blob/dev/bellows/exception.py#L1
